### DISCUSSION TITLE
What's the difference between Torus and Web3auth?

### DISCUSSION THREAD

**Discussion by nakulmk** [Web3Auth/Web3Auth#327] [Help]
What's the difference between Torus and Web3auth?

**Comment by YZhenY** [comment]
It is important to note that Web3Auth is not a wallet, it is wallet infrastructure that is plugged into applications or wallets. Web3Auth a form of key management, similar to seed phrases, that results in a standard cryptographic key provider specific to the user and application.

Torus is a wallet that uses Web3Auth.

You can read more about it at https://web3auth.io/docs/overview/web3auth-for-wallets and https://web3auth.io/docs/overview/web3auth-for-dapps

**Reply by vimal691** [reply]
cannot see the link from outside web3auth team

**Reply by shahbaz17** [reply]
Hey @vimal691 

Update the link. Please see now.
